neuromation/api/parser.py

```diff
@@ -39,6 +53,25 @@ def volume(self, volume: str) -> Volume:
             storage_uri=storage_uri, container_path=container_path, read_only=read_only
         )
 
+    def build_secret_files(self, input_volumes: Set[str]) -> Set[SecretFile]:
```

Comment:
Here and below: the api call looks like `client.parse.build_secret_files(...)`. 
`parse.build_xxx()` looks weird, please choose another name for the method.

Reply:
Another problem is the method signature: it returns a set of secret files that should be extracted from `input_volumes` at the next step. 
It is not very handy and hard to explain (try to write lucid API docs as the exercise).
I can suggest another API: `client.parse.volumes()` that returns a named tuple of regular and secret volume sequences, e.g. `volumes, secret_files = client.parse.volumes(inp_vols)`
What do you think?

Reply:
I really like this clean approach, but parsing (non-secret) volumes with the `--ALL` option, which is implemented in `_build_volumes` seems to be fairly complicated, as it queries user's ACL. Do you think `Parser` should do that or how would you address this particular option for volumes?

Reply:
We can handle `"ALL"` on the CLI side but parse all other values in API parser.